src/components/genreSelect/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { GenreSelectProps } from './types';
+
+export default function GenreSelect({ genreNames, selectedGenre, onSelect } :GenreSelectProps) {
```

Comment:
When you're using function expression for components declaration, it's better to add return type `React.ReactNode` to be sure that you return proper value. https://echobind.com/post/react-with-typescript-components-as-function-declarations-vs-function-expressions
I prefere lambas, but I just read that this approach has its own drawbacks - https://medium.com/raccoons-group/why-you-probably-shouldnt-use-react-fc-to-type-your-react-components-37ca1243dd13. 
So, it's up to, just try to protect your self from returning a wrong value from your component unexpectedly.

Reply:
Thanks, will go with lambdas